llvm/test/Transforms/AggressiveInstCombine/strncmp-1.ll

```diff
@@ -0,0 +1,216 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 4
+; RUN: opt -S -passes=aggressive-instcombine -strncmp-inline-threshold=3 < %s | FileCheck %s
+
+; check whether we generate the right IR
+
+declare i32 @strncmp(ptr nocapture, ptr nocapture, i64)
+declare i32 @strcmp(ptr nocapture, ptr nocapture)
+
+@s2 = constant [2 x i8] c"a\00"
+@s3 = constant [3 x i8] c"ab\00"
+
+define i1 @test_strncmp_1(ptr nocapture readonly %s) {
```

Comment:
```suggestion
define i1 @test_strncmp_1(ptr %s) {
```
I don't think the attributes are relevant?

Reply:
done. These are removed.